components/ProductCard.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="card" data-testid="productCard">
+    <div class="product-labels">
+      <div v-for="label in product.productLabels" :key="label.sortKey">
+        <img :src="label.icon" />
+      </div>
+    </div>
+    <product-image
+      :url="product.url"
+      :image-url="product.images[0].url"
+      :title="imgAlt"
+    />
+    <splash :product="product" />
```

Comment:
@coopjakob you're right, I'll change that.

Yes, it's possible to add a class to `<splash>` like your first example. Both are acceptable, it's just where you want to put the styling (parent vs child component)

Reply:
The size and position of component should be set by the parent, and the styling should be set by the child.

But let me think.

It's better to wrap a div around it and give that a size and position because then it won't be like you're intruding a components settings. So class should only be used for overwriting styling like colors etc.

Oh, and we can even create a position component instead of using a div.